p2p/types/proto/p2p_config.proto

```diff
@@ -0,0 +1,16 @@
+syntax = "proto3";
+package p2p;
+
+option go_package = "github.com/pokt-network/pocket/p2p/types";
+
+message P2PConfig {
+  string private_key = 1;
+  uint32 consensus_port = 2;
```

Comment:
What is this parameter supposed to be?

Reply:
`bool use_rain_tree = 3;  // whether or not we use rain tree`
`bool is_empty_connection_type = 4; // if the connection type is using tcp or empty`

Reply:
NOTE: I didn't introduce new configurations, so keep in mind that any contention with the configuration might be out of scope